tests/Test.ADAL.NET.Unit/TokenCacheUnitTests.cs

```diff
@@ -66,5 +68,13 @@ public void TokenCacheSerializationTest()
         {
             TokenCacheTests.TokenCacheSerializationTest();
         }
-    }
+
+        [TestMethod]
+         [Description("Test for Token Cache backwasrd compatiblity where new attribute is added in AuthenticationResultEx")]
+         [TestCategory("AdalDotNetUnit")]
+         public void TokenCacheBackCompatTest()
+         {
+                 TokenCacheTests.TokenCacheBackCompatTest(File.ReadAllBytes("oldcache.txt"));
```

Comment:
Not being able to see the data that is actually being tested against in the code review makes me a sad panda.